Marlin/pins_PRINTRBOARD_REVF.h

```diff
@@ -117,17 +154,54 @@
 // Temperature Sensors
 //
 #define TEMP_0_PIN          1   // Analog Input (Extruder)
+#define TEMP_A_PIN          2   // Analog Input (Extrudrboard A THERM)
+#define TEMP_B_PIN          3   // Analog Input (Extrudrboard B THERM)
 #define TEMP_BED_PIN        0   // Analog Input (Bed)
 
 //
 // Heaters / Fans
 //
 #define HEATER_0_PIN       15   // C5 PWM3B - Extruder
-#define HEATER_1_PIN       44   // F6
-#define HEATER_2_PIN       45   // F7
+// Note that Printrboard REV F6 removed HEATER_A_PIN from EXP1 and
+// reassigned it for a second fan output.
+#define HEATER_A_PIN       44   // F6 - Extrudrboard A HOTEND
+#define HEATER_B_PIN       45   // F7 - Extrudrboard B HOTEND
 #define HEATER_BED_PIN     14   // C4 PWM3C
 
 #define FAN_PIN            16   // C6 PWM3A
+#define PRINTRBOARD_F6_HEATER_FAN_PIN 44 // pin F6 on REV F6, see above.
+
+#ifndef NO_EXTRUDRBOARD
```

Comment:
I am currently running this on my REV F5 with a stock extrudrboard and I've
checked this against the schematic for the REV F4 and REV F6 (available at
https://github.com/Printrbot/printrboard ).  I'm guessing you're patching Printrbot's fork, which still uses the old fastio pin numbering and that's why these don't match your configuration.  This is correct arduino numbering.